spec/spec_helper.rb

```diff
@@ -225,14 +225,18 @@ def with_hidden_and_successful_requests
 # Reset the default locale, making sure that the previous default locale
 # is also cleared from the fallbacks
 def with_default_locale(locale)
-  original_default_locale = I18n.default_locale
-  original_fallbacks = I18n.fallbacks
-  I18n.fallbacks = nil
-  I18n.default_locale = locale
+  original_default_locale = AlaveteliLocalization.default_locale
+  original_fallbacks = AlaveteliLocalization.available_locales.join(" ")
+  AlaveteliLocalization.set_locales(
+      locale.to_s,
```

Comment:
Oh, [it does](https://github.com/mysociety/alaveteli/blob/54c139ba6e258d1c07a36a24f28fb703454364f1/lib/alaveteli_localization.rb#L7-L11) for `available_locales`.

Looks like this could be:

```ruby
AlaveteliLocalization.set_locales(locale, locale)
```